docs/content/installation/index.md

```diff
@@ -10,6 +10,9 @@ Radius consists of a set of tools and services that together form the Radius pla
 
 {{< image src="radius.png" alt="Diagram showing rad CLI and VSCode extension on local machine plus the Radius control plane on a Kubernetes cluster" width="600px" >}}
 
+## Prerequisites
+For edge version, Oras installation is required: https://oras.land/docs/installation.
+
 ## Step 1: Install the rad CLI
 
 {{< read file= "/shared-content/installation/rad-cli/install-rad-cli.md" >}}
```

Comment:
Will remove it from step 1 https://github.com/radius-project/docs/blob/edge/docs/shared-content/installation/rad-cli/install-rad-cli.md?plain=1#L12 in next commit. Direct edit link on the docs website doesn't allow updating multiple files at once.